### PR TITLE
New memcached adapter

### PR DESCRIPTION
**Pull request by vlas**
New Memcached adapter:
- does not support flush of expired (flush removes all)

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+jsonpickle==3.0.1
```
Comment (othercodes) [inline requirements/dev.txt]: why requirements.txt? pyproject is the same

File: rndi/cache/adapters/memcached/adapter.py [hunk 2]
```diff
@@ -3,35 +3,112 @@
...
+            retry_delay=self.RETRY_DELAY,
+            retry_for=[
+                MemcacheUnexpectedCloseError, MemcacheUnknownError, MemcacheServerError,
+                MemcacheUnknownCommandError, MemcacheClientError,
+                socket.timeout, socket.error,
+            ],
+        )
+
+    @property
+    def client(self):
```
Comment (sergiopalacio) [inline rndi/cache/adapters/memcached/adapter.py]: Hi! the adapter should only expose the public methods specified in the interface. 
I think its enough to have the protected properties as you did with self._client and self._ttl, the code that will consume this adapter will be through an interface, so they will never use client and ttl properties.

Comment (vlas) [reply rndi/cache/adapters/memcached/adapter.py]: makes sense, thanks. Updated

File: requirements/dev.txt (hunk 1 above)
Comment (vlas) [reply requirements/dev.txt]: This was they way I got used to do, though could be done without. Updated PR to use only poetry configs

**Review by othercodes — APPROVED** [review]

**Pull request merged**